spec/util/extensions/miq-module_spec.rb

```diff
@@ -41,6 +41,22 @@
       expect(test_module.default).to eq(value)
     end
 
+    it 'will return a prior cached value if resetting the value fails' do
+      $already_run = false
```

Comment:
I hate this test but it recreates it... any ideas how to do this without a global variable?